django_lightweight_queue/worker.py

```diff
@@ -55,14 +56,23 @@ def run(self):
         backend = get_backend()
         self.log.info("Loaded backend %s", backend)
 
-        while self.running.value:
+        time_item_last_processed = datetime.datetime.now()
```

Comment:
I don't like using real-time clocks for counting. This might have *really* weird behaviour around the BST/GMT switch, for instance.

Is there a good equivalent to `time.monotonic` we could use?